.github/workflows/auto-assign-to-project.yml

```
@@ -0,0 +1,26 @@
+name: Auto Assign to Project(s)
+
+on:
+  pull_request:
+    types: [opened, labeled]
+env:
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

It seems that you must name this environment variable as `MY_GITHUB_TOKEN`. You can see it at https://github.com/srggrs/assign-one-project-github-action#organisation-or-user-project

A bit weird the name :sweat_smile: 

```suggestion
  MY_GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

---

Yes, you're right 😅